WORKSPACE

```diff
@@ -42,10 +42,34 @@ git_repository(
     tag = "release-1.12.1",
 )
 
+# We use a patched version of google/benchmark/BUILD.bazel to keep the name
+# used to refer to libpfm4 and its targets consistent with other dependencies.
 git_repository(
     name = "com_github_google_benchmark",
+    patch_args = ["-p1"],
+    patches = ["@//:benchmark_build.patch"],
     remote = "https://github.com/google/benchmark.git",
-    tag = "v1.8.0",
+    tag = "v1.8.2",
+)
+
+# rules_foreign_cc is required to build libpfm4 since it is originally
+# configured to be built using GNU Make.
+git_repository(
+    name = "rules_foreign_cc",
+    commit = "816905a078773405803e86635def78b61d2f782d",
+    remote = "https://github.com/bazelbuild/rules_foreign_cc.git",
+)
+
+load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
+
+rules_foreign_cc_dependencies()
+
```

Comment:
I think we don't need this anymore.

Reply:
The build fails without it - I had actually mistakenly removed this and `rules_foreign_cc` in the commit before this one, thinking it was no longer needed since building `libpfm4` had been handed off to the LLVM third party rules, but it was only actually working because I hadn't `bazel clean --expunge`d before making sure it would build.

Reply:
Ack. In that case, please add a comment that it is needed to build libpfm.